app/models/manageiq/providers/ibm_cloud/power_virtual_servers/cloud_manager/placement_group.rb

```diff
@@ -0,0 +1,10 @@
+class ManageIQ::Providers::IbmCloud::PowerVirtualServers::CloudManager::PlacementGroup < ::PlacementGroup
+  supports :create
+  supports :delete
+
+  def self.raw_create_placement_group(ext_management_system, create_options)
+  end
+
+  def raw_delete_placement_group
+  end
```

Comment:
I know we're trying to get the refresher specs to pass, but if we're not going to implement this yet we shouldn't claim support
```suggestion
```

Reply:
definitely. Let me finish refresher spec pass.